db/migrate/1_create_pay_tables.rb

```diff
@@ -12,8 +15,8 @@ def change
     add_index :pay_customers, [:owner_type, :owner_id, :deleted_at, :default], name: :pay_customer_owner_index
     add_index :pay_customers, [:processor, :processor_id], unique: true
 
-    create_table :pay_merchants do |t|
-      t.belongs_to :owner, polymorphic: true, index: false
+    create_table :pay_merchants, id: :uuid do |t|
```

Comment:
```suggestion
    create_table :pay_merchants, id: primary_key_type do |t|
```

These should use the `primary_key_type` variable, right?

Reply:
🙈 Fixed!